toolchain/check/context.cpp

```diff
@@ -505,6 +532,102 @@ auto Context::ImplicitAsRequired(Parse::Node parse_node, SemIR::NodeId value_id,
   return output_value_id;
 }
 
+static auto CanConvertTupleToArray(Context& context, SemIR::Node tuple_type,
+                                   SemIR::Node array_type) -> bool {
```

Comment:
Maybe have this return the enum directly, avoiding the ternary at the caller?

Reply:
Obsolete after #3208.